Backoffice/Collector/FrontRoleCollector.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\Collector;
+
+use FOS\UserBundle\Model\UserInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Class FrontRoleCollector
+ */
+class FrontRoleCollector implements RoleCollectorInterface
+{
+    protected $roles = array();
+    protected $translator;
+
+    /**
+     * @param TranslatorInterface $translator
+     */
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    /**
+     * @return array
+     */
+    public function getRoles()
+    {
+        $this->roles[UserInterface::ROLE_DEFAULT] = $this->translator->trans(
```

Comment:
why add this role in this method and not in the constructor or in the compiler pass ?